src/components/LanguageSelector/index.less

```diff
@@ -0,0 +1,4 @@
+.languageSelect {
```

Comment:
```suggestion
.language-select {
```

Reply:
we usually format css classes like this. if you accept this, you'll also have to adjust the `LanguageSelect` component though